lib/ibmdb.js

```diff
@@ -757,18 +765,11 @@ IBMDB.prototype.buildIndexes = function(model) {
                       ' (\"' + columns + '\");\n');
   }
 
-  // Define index for each of the properties
-  // for (var p in definition.properties) {
-  //   var propIndex = this.buildIndex(model, p);
-  //   if (propIndex) {
-  //     indexClauses.push(propIndex);
-  //   }
-  // }
-
   return indexClauses;
 };
 
 IBMDB.prototype.buildColumnDefinition = function(model, prop) {
+  debug('IBMDB.prototype.buildColumnDefinition');
```

Comment:
@qpresley 
>I haven't back fit to 2.x yet so will need some assistance with that.

That's how I would do it:
```
# Go to 2.x branch
git checkout loopback-2.x
# create a feature branch
git checkout -b backport/add-debig-2.x
git cherry-pick <sha>
git push origin backport/add-debig-2.x -u
```
You can get the `sha` key of you commit after landing this PR and going to https://github.com/strongloop/loopback-ibmdb/commits/master, then you would be able to copy the `sha` key of the commit you want to backport; please see:

![screen shot 2016-09-28 at 12 24 37 pm](https://cloud.githubusercontent.com/assets/10672146/18922472/c2508a10-8576-11e6-9683-b8a83b434fd5.png)


I hope thats clear... Please let me know if you need more help/info or want me to do it on my end?

Thanks!